### ISSUE TITLE
Focus highlights for round buttons are not centered (Discrete screen)

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/fourier-making-waves#89]
Reported by @arouinfar in https://github.com/phetsims/fourier-making-waves/issues/68#issuecomment-862927885, for the Discrete screen:

One odd thing I noticed was that in my local copy (pulled all late evening 6/15) was that the focus on the round buttons on the Discrete screen was askew, but looked fine on the other two screens. For example:
<img width="57" alt="Screen Shot 2021-06-16 at 8 30 34 AM" src="https://user-images.githubusercontent.com/8419308/122334673-5304ce80-cef7-11eb-9e51-3a2cefd94f33.png">
<img width="100" alt="Screen Shot 2021-06-16 at 8 33 38 AM" src="https://user-images.githubusercontent.com/8419308/122334703-5d26cd00-cef7-11eb-9d1a-9b332894b401.png">

**Comment by pixelzoom** [comment]
@arouinfar OS and browser versions?

**Comment by pixelzoom** [comment]
I cannot reproduce this with 1.0.0-dev.29 or master, using macOS 11.3.1 + Safari 14.1 + Chrome 91.0.4472.106. 

@arouinfar does this behavior change if you resize your browser window?  If wonder if there's some rounding error in the computation of the radius for the focus highlight.

**Comment by pixelzoom** [comment]
Here's the code in RoundButton that controls the focus highlight.  I confirm that this is the correct code  by changing `5` to a larger number.

```js
    // pdom - focus highlight is circular for round buttons, with a little bit of padding
    // between button shape and inner edge of highlight
    this.focusHighlight = Shape.circle( 0, 0, buttonBackgroundRadius + 5 );
```

This was implemented by @jessegreenberg in https://github.com/phetsims/sun/commit/e911a9aa969fe60e95d6822931935d64e4d5947d back in 2017.  It hasn't changed since then. (RoundButtonView was renamed to RoundButton in unrelated work.)

@jessegreenberg Can you think of any reason why the focus highlight would look off-center (and not be consistently reproducible) as shown in @arouinfar's screenshots above?

**Comment by jessegreenberg** [comment]
I was not able to get this to happen on my main Windows development machine, but I happened to try it on a laptop that I had set to a an odd screen resolution (1360x788) and I saw this right away on Windows Chrome. When I zoomed into the button, the problem went away as the button and highlight got larger. Then, when reset to recommended device resolution (3840x2160) the problem went away again. So now I am thinking it could be something deeper in scenery instead of round buttons. Could be related to rounding making it more noticeable on lower resolutions.

**Comment by jessegreenberg** [comment]
@jonathanolson and I met today and he helped investigate this. Unfortunately, we were not able to see this issue in the same way on my machine in Chrome, even when I had it set to lower resolutions. I had not pulled since I last investigated in https://github.com/phetsims/fourier-making-waves/issues/89#issuecomment-863607870. We thought Zoom screen share might be interfering so I tried again without using Zoom (resetting my computer between attempts), but still didn't see this.

We saw this to some extent on Windows Firefox at very low screen resolutions/screen sizes. When zooming into the sim on Firefox, simulation components kind of jostle around during animation but they stop moving at the end. But the focus highlight does NOT jostle with the sim components. At the end of animation the highlight may be off center. @jonathanolson explained the jostling is likely due to a Firefox optimization. Screen capture of this using Firefox at 800x600 screen resolution:

<details>

![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/6396244/122588457-1dfa9800-d02d-11eb-9a11-be975462b5f9.gif)

</details>

We think highlights don't jostle with the sim is because they are on a different Display. To verify, we added a circle around the ResetAllButton in the Discrete screen, and saw that even with jostle the circle was always centered around the button. This could likely be fixed by changing the focus highlight implementation to use the same Display as the sim. But if we spend time on this it may be worth investigating if we can prevent the jostle in the first place.

I don't know if either of these things are worth doing right now. @arouinfar can you please comment on how easy it is for you to get this to happen? What platform are you using, and what is your screen resolution and browser window size? So far, I am not seeing this during typical usage so it may not be worth spending time at the moment.

**Comment by pixelzoom** [comment]
This issue has stalled.  @jessegreenberg is waiting for this feedback from @arouinfar:

> @arouinfar can you please comment on how easy it is for you to get this to happen? What platform are you using, and what is your screen resolution and browser window size?

**Comment by pixelzoom** [comment]
This is occurring with ResetAllButton and InfoButton, common code components.  And Fourier is using their default pointer areas and focus highlights.  So if this is not blocking for Fourier, then I'd like to transfer this issue to a more general repo.
